language/stdlib/modules/LibraTimestamp.move

```diff
@@ -47,6 +48,7 @@ module LibraTimestamp {
             exists<CurrentTimeMicroseconds>(CoreAddresses::LIBRA_ROOT_ADDRESS()) && now_microseconds() == 0,
             ETIME_NOT_INITIALIZED
         );
+        assert(!exists<TimeHasStarted>(Signer::address_of(lr_account)), ETIME_ALREADY_STARTED);
```

Comment:
Could we use `EGENESIS_ONLY` (or equivalent) here, or is this a different problem?

Reply:
I like EGENESIS_ONLY, and it makes sense here. I'll update.